README.md

```diff
@@ -98,31 +98,77 @@ cert-manager-webhook-68d464c8b-hvpf6       1/1     Running   0          33s
 
 ### Install KintoHub
 
-Run
+Run the following steps depending on the setup you want.  
+Each one of the following steps are "incremental", you must run all the steps prior to the setup your chose.
+
+- **Generic Args**
+
+  ```sh
+  export KINTO_ARGS="--set minio.resources.requests.memory=null \
+  --set minio.makeBucketJob.resources.requests.memory=null"
+  ```
+
+- **KintoHub Local**
+
+  KintoHub is installed on a local cluster with no inbound internet connection.
+
+  ```sh
+  export KINTO_ARGS="${KINTO_ARGS} \
+  --set builder.env.IMAGE_REGISTRY_HOST=kintohub \
+  --set builder.workflow.docker.registry=https://index.docker.io/v1/ \
+  --set builder.workflow.docker.email=devaccounts@kintohub.com \
+  --set builder.workflow.docker.username=docker-username \
+  --set builder.workflow.docker.password=docker-password"
+  ```
+
+  Notes: change the parameters above.
+
+- **Kinto Dashboard Local but Web Services Exposed Publicly _WITHOUT_ SSL**
+
+  Kintohub Dashboard is not publicly exposed, you will need a port-forward to access it.  
+  However, the web services deployed using KintoHub are exposed without SSL.
+
+  ```sh
+  export KINTO_ARGS="${KINTO_ARGS} \
+  --set nginx-ingress-controller.service.type=LoadBalancer \
+  --set common.domainName=oss.kintohub.net"
+  ```
+
+  Notes: change the parameters above.
+
+- **KintoHub Dashboard Local but Web Services Exposed Publicly _WITH_ SSL**
```

Comment:
confused. this sounds mutually exclusive from the above one? but at the top you said it is incremental.